scripts/ci/run-test-acceptance-tests.sh

```diff
@@ -13,9 +17,9 @@ fi
 ${TESTCMD} --label=check-roslyn --timeout=60m make -C acceptance-tests check-roslyn
 
 # run CoreCLR managed tests, we precompile them in parallel so individual steps don't need to do it
-${TESTCMD} --label=coreclr-compile-tests --timeout=120m --fatal make -C acceptance-tests coreclr-compile-tests
-${TESTCMD} --label=coreclr-runtest-basic --timeout=10m make -C acceptance-tests coreclr-runtest-basic
+${TESTCMD} --label=coreclr-compile-tests --timeout=140m --fatal make -C acceptance-tests coreclr-compile-tests
+${TESTCMD} --label=coreclr-runtest-basic --timeout=20m make -C acceptance-tests coreclr-runtest-basic
 ${TESTCMD} --label=coreclr-runtest-coremanglib --timeout=10m make -C acceptance-tests coreclr-runtest-coremanglib
 
 # run the GC stress tests (on PRs we only run a short version)
-${TESTCMD} --label=coreclr-gcstress --timeout=1200m make -C acceptance-tests coreclr-gcstress CI_PR=${ghprbPullId}
+${TESTCMD} --label=coreclr-gcstress --timeout=40m make -C acceptance-tests coreclr-gcstress CI_PR=${ghprbPullId}
```

Comment:
this won't work, the GC stress tests take about 15 hours by design on non-PR runs. You'll need to make it conditional on whether `${ghprbPullId}` is set.